.github/workflows/deploy.yml

```
@@ -0,0 +1,77 @@
+      - name: 🔨 Build Game (Production)
+        run: npm run build
+
+      - name: ✅ Verify Build Output
+        run: |
+          echo "Verifying build artifacts..."
+          test -f dist/index.html || { echo "❌ index.html missing"; (exit 1); }
+          test -f dist/bundle.min.js || { echo "❌ bundle.min.js missing"; (exit 1); }
+          echo "✅ Build verified successfully!"
+
+      - name: 📂 Upload Pages Artifact
+        if: github.event_name == "push" && github.ref == "refs/heads/main"
```

actionlint (1.7.11)

[error] got unexpected character '"' while lexing expression, expecting 'a'..'z', 'A'..'Z', '_', '0'..'9', ''', '}', '(', ')', '[', ']', '.', '!', '<', '>', '=', '&', '|', '*', ',', ' '. do you mean string literals? only single quotes are available for string delimiter

(expression)